en/mavgen_c/README.md

```diff
@@ -52,6 +52,15 @@ When compiling the project, we recommend that you specify the top-level output d
 $ gcc ... -I generated/include -I generated/include/common ...
 ```
 
+In order to *send messages* you will also need to declare a variable `mavlink_system` in the **global scope**, specifying the system id and component ID of your component:
+```cpp
+mavlink_system_t mavlink_system = {
+	1, //Sytem ID (1-255)
+	1  //Component ID 
```

Comment:
If you're putting a range on system id then put a range here, too ('though it should be obvious from the type).

Reply:
I added clarification that this must be  of type MAV_COMPONENT - which is probably even more useful. Thanks